jdaviz/tests/test_viewer_ids.py

```diff
@@ -11,9 +11,9 @@ def test_default_viewer_ids_default():
 
 def test_default_viewer_ids_cubeviz(cubeviz_helper):
     x = cubeviz_helper.app
-    assert x.get_viewer_reference_names() == ['flux-viewer', 'uncert-viewer', 'mask-viewer',
+    assert x.get_viewer_reference_names() == ['flux-viewer', 'uncert-viewer',
                                               'spectrum-viewer']
-    assert x.get_viewer_ids() == ['cubeviz-0', 'cubeviz-1', 'cubeviz-2', 'cubeviz-3']
+    assert x.get_viewer_ids() == ['cubeviz-0', 'cubeviz-1', 'cubeviz-2']

```

Comment:
Since CI passed, probably doesn't matter but cubeviz-2 used to be image viewer but now a spectrum viewer. Something to keep in mind.